pkg/hooks/gen.go

```diff
@@ -78,7 +78,7 @@ func GenerateHooks(_ context.Context, cfg *config.Config, repo string) error {
 			Hook       string
 			Args       string
 		}{
-			Executable: ex,
```

Comment:
`ex` [here](https://github.com/charmbracelet/soft-serve/pull/459/files#diff-044c37c57751816374433ad91b45ccbf1063f64e1106e32b10a2ad1e97e56570R38-R44) is no longer used and should be removed.